values/vault-operator/vault-operator.gotmpl

```diff
@@ -5,3 +5,14 @@ global:
   imagePullSecrets:
     - otomi-pullsecret-global
 {{- end }}
+
+podSecurityContext:
+  runAsNonRoot: true
+  runAsUser: 65534
+  runAsGroup: 65534
+  fsGroup: 1337
+
+securityContext:
```

Comment:
if this targets the same pod (it seems it does), then `securityContext` is not needed when `podSecurityContext` is provided

Reply:
done